docs/workflow.md

```diff
@@ -0,0 +1,52 @@
+# Development Workflow
+
+## Fixing issues
+
+
+### Step 1: Pick an issue to fix
+ 
+ After selecting the issue
+ 
+  1.Comment on the issue saying you are working on the issue.
+
+  2.We except you to discuuss the approach either by commenting or in the gitter.
+  
+  3.Updates or progress on the issue would be nice.
+
+### Step 2: Branch policy<br>
+
+  Start off from your `development` branch and make sure it is up-to-date with the latest version of the committer repo's
+ `development` branch. Make sure you are working in development branch only.<br>
+ `git pull upstream development`
+    
+ If you have not added upstream follow the steps given [here](https://help.github.com/articles/configuring-a-remote-for-a-fork/).
+
+### Step 3: Coding Policy
+
```

Comment:
Code practices

* Please help us follow the best practice to make it easy for the reviewer as well as the contributor. We want to focus on the code quality more than on managing pull request ethics.
* Single commit per pull request
* Reference the issue numbers in the commit message. Follow the pattern Fixes #<issue number> <commit message>
* Follow uniform design practices. The design language must be consistent throughout the app.
* The pull request will not get merged until and unless the commits are squashed. In case there are multiple commits on the PR, the commit author needs to squash them and not the maintainers cherrypicking and merging squashes.
* If the PR is related to any front end change, please attach relevant screenshots in the pull request description.